torch/_export/serde/serialize.py

```diff
@@ -594,8 +594,8 @@ def export_nn_module_stack(val):
         if torch_fn := node.meta.get("torch_fn"):
             ret["torch_fn"] = ST_DELIMITER.join(list(torch_fn))
 
-        if quantization_tag := node.meta.get("quantization_tag"):
-            ret["quantization_tag"] = json.dumps(quantization_tag)
+        if custom := node.meta.get("custom"):
+            ret["custom"] = json.dumps(custom)
```

Comment:
Is custom object always json serializable? Should make this explicit in the error message

Reply:
yeah will do.